### ISSUE TITLE
[Request] File history window not showing merge commit changes

### ISSUE THREAD

**Issue by JohannesDeml** [ForkIssues/Tracker#1580]
Hey there,

I tried to find out when a particular line was removed through the File History window. However I only found when it was created, but not when it was deleted. Through manual searching, I found that the change happened in a merge commit that had a conflict in this file and was resolved the wrong way. 

Do you exclude merge commits on purpose? At least for me I was under the impression, that I would see all changes in the file history window and that idea would be more useful for me, than excluding merge commits.

MacOS BigSur 11.6.3
Fork 2.16

**Comment by DanPristupov** [comment]
> Do you exclude merge commits on purpose?

No. I'm not sure about the reason, but that's how git works: https://stackoverflow.com/a/35786883

**Comment by JohannesDeml** [comment]
Thank you for your quick answer! I almost feared, that this is something git-specific. Is there a chance you can find a solution on your side to support this in fork somehow? I really enjoy the history feature, but if I'm not able to track all commits that contain changes (especially if they happen in merge conflicts), then the feature loses quite a bit of value for me.

**Comment by ck625446567** [comment]
Same issue.

**Comment by JonUleis** [comment]
Really hoping this gets fixed, as the History view is incomplete when it's excluding edits that occurred during merges. 

Same issue is here on the Windows repo as well: https://github.com/fork-dev/TrackerWin/issues/1261

**Comment by goran-w** [comment]
FYI, in the "sibling" issue for Windows, I made an extensive comment listing the most relevant `git-log` option-flags for addressing this issue: https://github.com/fork-dev/TrackerWin/issues/1261#issuecomment-3738153416

**Comment by saintisk** [comment]
Same Problem.
MacOS / Fork 2.69.0
